packages/yarnpkg-pnp/sources/loader/makeManager.ts

```diff
@@ -93,7 +93,7 @@ export function makeManager(pnpapi: PnpApi, opts: MakeManagerOptions) {
 
     if (typeof parent.pnpApiPath === `undefined`) {
       if (parent.filename !== null) {
-        return findApiPathFor(parent.filename);
+        return parent.pnpApiPath = findApiPathFor(parent.filename);
```

Comment:
How come we have parents without `pnpApiPath`? It should be set here:

https://github.com/yarnpkg/berry/blob/5f0439a087b8463d97fb70c96dc6f983a91be512/packages/yarnpkg-pnp/sources/loader/applyPatch.ts#L135

Reply:
In this repo it's `yarnpkg-cli/sources/boot-cli-dev.js`

When running `yarn build` in https://github.com/SagnikPradhan/sagnikpradhan/tree/feat/yarn2 it's
```
src/@root
src/components/@root
src/routes/@root
```

When running in some vue projects at work it's `@root`